tests/sentry/integrations/github/test_integration.py

```diff
@@ -251,10 +251,8 @@ def test_basic_flow(self):
         assert integration.external_id == self.installation_id
         assert integration.name == "Test Organization"
         assert integration.metadata == {
-            "access_token": None,
-            # The metadata doesn't get saved with the timezone "Z" character
-            # for some reason, so just compare everything but that.
-            "expires_at": None,
+            "access_token": self.access_token,
+            "expires_at": self.expires_at,
```

Comment:
this change is expected. we are now calling another GitHub API in `post_install`, which calls [authorize_request](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/github/client.py#L137-L164) in GithubProxyClient and subsequently [_get_token](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/github/client.py#L97-L135).

the task calls `/installation/repositories` from the GitHub API, which requires an access token according to `_get_token`, which causes the integration's access token to be refreshed aka updated with not-None `access_token` and `expires_at` values.